GEECS-PythonAPI/geecs_python_api/controls/interface/geecs_errors.py

```
@@ -1,103 +1,112 @@
+"""Error definitions for geecs api.
+
+Note: ErrorAPI is being Deprecated in favor of calls to standard logging.
+"""
+
+from __future__ import annotations
+
 import logging
-from typing import Optional
+import warnings
+from typing import Optional, Tuple
 
-# Set up a module-level logger
 logger = logging.getLogger(__name__)
 
+
 class GeecsDeviceInstantiationError(Exception):
-    """Exception raised when a GEecs device fails to instantiate."""
-    def __init__(self, message: str):
-        super().__init__(message)
-        self.message = message
+    """Raised when a GEECS device fails to instantiate."""
```

I am confused about the changes of the whole GeecsDeviceInstantiationError class. Now, there is no code in this class anymore, isn't it? Shouldn't this should raise an error in itself?